apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/LoggingSnippets.cs

```diff
@@ -69,11 +70,18 @@ public async Task Logs_Information()
         [Fact]
         public async Task Logs_PropagateExceptions()
         {
-            using (TestServer server = new TestServer(GetExceptionPropagatingHostBuilder()))
-            using (HttpClient client = server.CreateClient())
+            var aggregateEx = await Assert.ThrowsAsync<AggregateException>(async () =>
             {
-                await Assert.ThrowsAsync<AggregateException>(() => client.GetAsync($"/LoggingSamples/LogInformation/{_testId}"));
-            }
+                using (TestServer server = new TestServer(GetExceptionPropagatingHostBuilder()))
+                using (HttpClient client = server.CreateClient())
+                {
+                    await client.GetAsync($"/LoggingSamples/LogInformation/{_testId}");
+                }
+            });
+
+            var rpcException = aggregateEx.InnerException as RpcException;
```

Comment:
I'd just cast to RpcException here. That way we'll get to see the actual exception type if it's wrong, rather than just Assert.NotNull failing.

Reply:
Yep, done.